### PR TITLE
added new conversion article

### PR DESCRIPTION
**Pull request by DKaramfilov**
245863 PdfProcessing: Add info about conversion from flow formats to PDF

### CODE REVIEW

**Review by Svilen-Pavlov — CHANGES_REQUESTED** [review]
Typos on lines 14 & 15
"More infroamtion is availble here"

File: libraries/radpdfprocessing/formats-and-conversion/converting-other_formats.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+published: True
+position: 3
+---
+
+# Converting other formats to PDF
+ 
+ Converting to PDF from other formats is supported but you need to use the other libraries from the suite. 
+ 
+ * __Converting Docx, Doc, RTF, and HTML:__ Use the WordsProcessing library. More infroamtion is availble here: [Formats and Convertion]({%slug radwordsprocessing-formats-and-conversion%}).
+ * __Converting Xlsx, Csv:__ Use the SpreadProcessing library. More information is availbel here:  [Using PdfFormatProvider]({%slug radspreadprocessing-formats-and-conversion-pdf-pdfformatprovider%}).
```
Comment (TanyaDimitrova) [inline libraries/radpdfprocessing/formats-and-conversion/converting-other_formats.md:15]: - More information is available here
- Add all supported formats

File: libraries/radpdfprocessing/formats-and-conversion/converting-other_formats.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+tags: pdf,docx,doc,rtf,xlsx,html
+published: True
+position: 3
+---
+
+# Converting other formats to PDF
+ 
+ Converting to PDF from other formats is supported but you need to use the other libraries from the suite. 
+ 
+ * __Converting Docx, Doc, RTF, and HTML:__ Use the WordsProcessing library. More infroamtion is availble here: [Formats and Convertion]({%slug radwordsprocessing-formats-and-conversion%}).
```
Comment (TanyaDimitrova) [inline libraries/radpdfprocessing/formats-and-conversion/converting-other_formats.md:14]: More information is available here:

File: libraries/radpdfprocessing/formats-and-conversion/converting-other_formats.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Converting other formats to PDF
+page_title: Converting other formats to PDF
+slug: radpdfprocessing-formats-and-conversion-converting-other-formats-to-pdf
+tags: pdf,docx,doc,rtf,xlsx,html
+published: True
+position: 3
+---
+
+# Converting other formats to PDF
```
Comment (TanyaDimitrova) [inline libraries/radpdfprocessing/formats-and-conversion/converting-other_formats.md:10]: Convert from Other Formats to PDF

**Review by TanyaDimitrova — CHANGES_REQUESTED** [review]

**Pull request merged**